fusi_bids_pydantic.py

```diff
@@ -59,6 +59,16 @@ class Hardware(BaseModel):
             alias="DeviceSerialNumber",
         )
     )
+    transducer_serial_number: Optional[str] = Field(
+        None,
+        description="The serial number of the  detachable ultrasound transducer that produced the measurements.",
+        alias="TransducerSerialNumber",
+    )
+    laptop_serial_number: Optional[str] = Field(
+        None,
+        description="The serial number of the laptop that produced the measurements.",
+        alias="LaptopSerialNumber",
+    )
```

Comment:
On second thought, let me verify my understanding of the BIDS spec; maybe these fields are kinda-covered or need to be renamed in the BIDS spec

asked a question to the maintainers: https://docs.google.com/document/d/1W3z01mf1E8cfg_OY7ZGqeUeOKv659jCHQBXavtmT-T8/edit?disco=AAABeF_X4qw

Reply:
Looks like Samuel agreed to 
- add the probe serial number separately
- and suggested that station name refer to the computer (eg artemis )

So we won't need these additional fields anymore